packages/toolkit/src/lib/vdp-sdk/model/mutations.ts

```diff
@@ -36,6 +36,16 @@ export type CreateUserModelPayload = {
   configuration: Record<string, string>;
 };
 
+export type UpdateUserModelPayload = {
+  description?: string;
+  source_url?: string;
+  documentation_url?: string;
+  license?: string;
+  //visibility: Visibility;
```

Comment:
@orangecoloured What happened with the visibility?

Reply:
It can't be changed yet via `PATCH`. @heiruwu said that it's gonna change soon and we'll put it back there.

Reply:
got it!! Thanks for clarification